lib/helpers.bash

```diff
@@ -105,6 +105,8 @@ bash-it ()
     example '$ bash-it migrate'
     example '$ bash-it update'
     example '$ bash-it search [-|@]term1 [-|@]term2 ... [ -e/--enable ] [ -d/--disable ] [ -r/--refresh ] [ -c/--no-color ]'
+	example '$ bash-it preview'
```

Comment:
sorry about the nit-pick , but it seems like the spacing here is not like in the rest of the file 😢

Reply:
Until my `helpers` branch is committed, `lib/helpers` has not been through `shfmt` so the spacing varies between functions even...